ml-agents/mlagents/trainers/policy/torch_policy.py

```diff
@@ -114,7 +114,7 @@ def update_normalization(self, vector_obs: np.ndarray) -> None:
         If this policy normalizes vector observations, this will update the norm values in the graph.
         :param vector_obs: The vector observations to add to the running estimate of the distribution.
         """
-        vector_obs = [torch.as_tensor(vector_obs)]
+        vector_obs = [torch.as_tensor(vector_obs)[:, 1:]]
```

Comment:
We should be able to use the BehaviorSpec to get the list of SensorTypes, and use that to separate out goals from observations.